.golangci.yaml

```diff
@@ -0,0 +1,92 @@
+# todo: this should match the configuration of osv-scanner for the most part
```

Comment:
Create an issue for TODOs and link to the issue in the comment.

Reply:
Should be good to go once the other PRs are merged in (I'll wait with approval until then to make internal merging easier)